support/documentation/architecture/decisions/0003-use-lombok.md

```diff
@@ -0,0 +1,40 @@
+# Use Lombok
+
+- Status: accepted
+- Date: 2020-05-11
+
+## Context and Problem Statement
+
+Java introduces a lot of boilerplate code that reduces readability of code. How can we reduce that boilerplate code?
+
+## Decision Outcome
```

Comment:
No `Considered options` here. That's probably ok but just wanted to check if we had any?

Reply:
Yes, didn't actually consider different options here, so thats why it is empty.